src/MetaMorpho.sol

```diff
@@ -294,7 +294,7 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
     function submitMarketRemoval(MarketParams memory marketParams) external onlyCuratorRole {
         Id id = marketParams.id();
         if (config[id].removableAt != 0) revert ErrorsLib.AlreadyPending();
-        if (pendingCap[id].validAt != 0) revert ErrorsLib.PendingCap();
+        if (pendingCap[id].validAt != 0) revert ErrorsLib.PendingCap(id);
```

Comment:
For a better readability, it would be better to have all the `config` checks done one after another and the `pendingCap` one at the very end.

Also, it should avoid an `SLOAD` if something reverts because of the checks done within the `config`

Reply:
Indeed we could, even if it's not a big deal IMO

Reply:
You know... it's a nitpick 😎